sample/db/samples/payment_methods.rb

```diff
@@ -2,10 +2,14 @@
   name: 'Credit Card',
   description: 'Bogus payment gateway.',
   active: true
-).first_or_create!
+).first_or_create! do |payment_method|
+  payment_method.store_ids = Spree::Store.ids
+end
 
 Spree::PaymentMethod::Check.where(
   name: 'Check',
   description: 'Pay by check.',
   active: true
-).first_or_create!
+).first_or_create! do |payment_method|
```

Comment:
@MatthewKennedy, unfortunately, this will not work, we need to change this to eg.

```ruby
cc_payment_method = Spree::Gateway::Bogus.where(
  name: 'Credit Card',
  description: 'Bogus payment gateway.',
  active: true
).first_or_create!

cc_payment_method.store_ids = Spree::Store.ids
cc_payment_method.save!
```

Reply:
Updated.